Description: stop linking library with libpcsclite
 Internally the library doesn't use libpcsclite. Stop using includes
 from libpcsclite-dev and linking with -lpcsclite
Author: Dmitry Eremin-Solenikov <dbaryshkov@gmail.com>

---

--- globalplatform-6.0.0.orig/configure.in
+++ globalplatform-6.0.0/configure.in
@@ -119,7 +119,7 @@ case "$host" in
   ;;
 *)
 	LDFLAGS="$LDFLAGS $PCSCLITE_LIBS"
-	AC_CHECK_LIB(pcsclite, SCardEstablishContext, [],
+	AC_CHECK_LIB(pcsclite, SCardEstablishContext, [:],
         [AC_MSG_ERROR([SCardEstablishContext() not found, install
 pcsc-lite 1.2.9-beta7 or later])])
   ;;
--- globalplatform-6.0.0.orig/src/Makefile.am
+++ globalplatform-6.0.0/src/Makefile.am
@@ -19,8 +19,8 @@ unzip/unzip.c unzip/unzip.h unzip/zip.c
 globalplatform/errorcodes.h globalplatform/stringify.h globalplatform/types.h globalplatform/error.h \
 dyn_generic.h dyn_unix.c connection.c stringify.c crypto.h debug.c util.c util.h loadfile.c loadfile.h crypto.c
 
-libglobalplatform_la_LDFLAGS = -version-info 6:0:0 $(PCSCLITE_LIBS)
-libglobalplatform_la_CPPFLAGS = $(AM_CPPFLAGS) $(PCSCLITE_CFLAGS)
+libglobalplatform_la_LDFLAGS = -version-info 6:0:0
+libglobalplatform_la_CPPFLAGS = $(AM_CPPFLAGS)
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = globalplatform.pc
--- globalplatform-6.0.0.orig/src/globalplatform/connection.h
+++ globalplatform-6.0.0/src/globalplatform/connection.h
@@ -29,11 +29,6 @@ extern "C"
 #endif
 
 #include <stdio.h>
-#ifdef WIN32
-#include <WinSCard.h>
-#else
-#include <PCSC/winscard.h>
-#endif
 #include "types.h"
 #include "library.h"
 #include "security.h"
--- globalplatform-6.0.0.orig/src/globalplatform.pc.in
+++ globalplatform-6.0.0/src/globalplatform.pc.in
@@ -6,6 +6,5 @@ includedir=@includedir@/globalplatform
 Name: GlobalPlatform Library
 Description: GlobalPlatform Smart Card Management Library
 Version: @VERSION@
-Requires: libpcsclite >= 1.2.7
 Libs: -L${libdir} -lglobalplatform
 Cflags: -I${includedir}
